portfolio/src/main/webapp/script.js

```diff
@@ -25,3 +28,11 @@ function prevImage(){
     currentIndex = (currentIndex + pictures.length - 1) % pictures.length;
     document.getElementById("projectImg").src = pictures[currentIndex];
 }
+
+function autoChange (){
+    currentIndex2 ++;
+    currentIndex2 %= mePics.length;
```

Comment:
Interesting that you opted for slightly different code here than in nextImage. Usually it's better to either keep to the same pattern or upgrade the old code.